src/rust/ide/src/model/suggestion_database/entry.rs

```diff
@@ -264,6 +264,16 @@ impl Entry {
                 kind          : Kind::Local,
                 scope         : Scope::InModule {range:scope.into()},
             },
+            Module {module,documentation,..} => Self {
+                name          : module.clone(),
+                arguments     : default(),
+                module        : module.try_into()?,
+                self_type     : None,
+                documentation,
+                kind          : Kind::Module,
+                scope         : Scope::Everywhere,
+                return_type   : "".to_string()
```

Comment:
In Enso module is a type of its own, so logically better will be put the `module` clone here.